e2e/desktop-layout-component.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { test, expect } from '@playwright/test';
+
+test.use({ storageState: 'playwright/.auth/certified-user.json' });
+
+test.describe('Classic challengs - 3 pane desktop layout component', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(
+      'learn/2022/responsive-web-design/build-a-survey-form-project/build-a-survey-form'
+    );
+  });
```

Comment:
There is a single test in this `describe` block, `beforeEach` hook is not really needed. It's ok if you decide  to leave it.

Reply:
Removed the beforeEach hook with [825a9a9](https://github.com/freeCodeCamp/freeCodeCamp/pull/52012/commits/825a9a9c856598bfbf4b9232618dbec93f64967f)